lib/nimble_template/addons/variants/phoenix/web/wallaby.ex

```diff
@@ -29,12 +31,17 @@ defmodule NimbleTemplate.Addons.Phoenix.Web.Wallaby do
   end
 
   defp copy_files(
-         %Project{web_module: web_module, base_module: base_module, web_test_path: web_test_path} =
-           project
+         %Project{
+           web_module: web_module,
+           base_module: base_module,
+           web_test_path: web_test_path,
+           installed_addons: installed_addons
+         } = project
        ) do
     binding = [
       web_module: web_module,
-      base_module: base_module
+      base_module: base_module,
+      with_ex_vcr?: ExVCR in installed_addons
```

Comment:
Could it be the situation when one module depends on multiple installed addons? So, maybe we can make `installed_addons` a separate module with methods to check enabled addons instead of using them as an array.

What do you think?

Reply:
Hi @Nihisil could you clarify your idea a bit are you suggesting to

1/ Change the `installed_addons`'s implementation?
2/ Change the `ExVCR in installed_addons` logic?

Reply:
I suggested both. We can have something like this

```
defmodule InstalledAddons do
  def add_addon() ...
  def is_ex_vcr_added?() ...
end
```

With that, we will encapsulate logic in one place, and if some addons would require more checks: for example, it may depend on multiple addons and rely on some addons properties.

Reply:
and might I know the logic for `is_ex_vcr_added?()` can we still reuse `ExVCR in installed_addons` 🤔

```elixir
defmodule InstalledAddons do
  def add_addon() ...
  
  def is_ex_vcr_added? do
    ExVCR in installed_addons
  end
end
```

Reply:
Yep, it is correct. My point for these changes is to have all the `is_...` methods in one place instead of using `ExVCR in installed_addons` across the whole code base.

What do you think?

Reply:
That is much better for sure, will find sometime to refactor this :D

Reply:
@andyduong1920 How about we do refactoring after [this RFC](https://github.com/nimblehq/elixir-templates/discussions/261) (https://github.com/nimblehq/elixir-templates/discussions/261) is finalized? so the part to check whether add-ons are added or not can utilize the changes from the RFC.

Reply:
Yeah that works as well @hanam1ni cc @Nihisil